encord/orm/dataset.py

```diff
@@ -297,7 +297,7 @@ def duration(self) -> Optional[int]:
     def client_metadata(self) -> Optional[dict]:
         """
         The currently cached client metadata. To cache the client metadata, use the
-        :meth:`~encord.orm.dataset.DataRow.fetch()` function.
+        :meth:`~encord.orm.dataset.DataRow._data()` function.
```

Comment:
looks off? why not `refetch_data()`?

Reply:
Huh, I think I accidentally deleted the refetch part when I was typing!